tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -82,6 +83,24 @@ def create_parser(product_choices=None):
     mode_group.add_argument("--verify-log-full", action="store_true",
                             default=False,
                             help="Output per-iteration test results when running verify")
+    mode_group.add_argument("--verify-repeat-loop", action="store",
+                            default=10,
+                            help="Run tests in a loop for a number of times",
+                            type=int)
+    mode_group.add_argument("--verify-repeat-restart", action="store",
+                            default=5,
+                            help="Restart the whole process",
+                            type=int)
+    mode_group.add_argument("--verify-no-chaos-mode", action="store_false",
+                            default=True,
+                            help="Disable chaos mode when running on Firefox")
+    mode_group.add_argument("--verify-max-time", action="store",
+                            default=None,
+                            help="The maximum number of minutes for each test to run",
```

Comment:
This isn't a time in minutes, it's in seconds. And also the time is the maximum time for the job to run, it's not a time per test.

Reply:
I've set it to a time in minutes:
https://github.com/w3c/web-platform-tests/blob/82da4d2663627032f6befd1a9618de51f31a2509/tools/wptrunner/wptrunner/wptcommandline.py#L100
Would you rather I change it to seconds?

Also, I'll fix the help string.

Reply:
Oh. Let's try this way first.